lib/note-detail.jsx

```diff
@@ -99,14 +104,36 @@ export default React.createClass( {
 						style={ divStyle }
 					>
 						<NoteContentEditor
-							ref={ this.saveEditorRef }
 							content={ content }
 							filter={ filter }
 							onChangeContent={ this.queueNoteSave }
+							ref={ this.saveEditorRef }
 						/>
 					</div>
 				) }
 			</div>
 		);
 	},
 } );
+
+const mapStateToProps = ( {
+	appState: state,
+	revision: { selectedRevision },
+	settings: { fontSize },
+} ) => {
+	const revision = selectedRevision || getNote( state );
+	const previewingMarkdown = get( revision, 'data.systemTags', '' ).indexOf( 'markdown' ) !== -1
+		&& state.editorMode === 'markdown';
```

Comment:
this is a weird dangle, also how about using something more semantic like `includes( get( revision, 'data.systemTags', '' ), 'markdown' )`

we could extract this into bindings or use parenthesis to clean it all up…

```js
const isMarkdownNote = includes( get( revision, 'data.systemTags', '' ), 'markdown' );
const isPreviewing = isMarkdownNote && state.editorMode == 'markdown';
```